addons/mass_mailing/models/mailing_list.py

```diff
@@ -48,9 +48,9 @@ def _compute_mailing_list_count(self):
         data = {}
         if self.ids:
             self.env.cr.execute('''
-                SELECT mailing_list_id, count(*)
-                FROM mail_mass_mailing_list_rel
-                WHERE mailing_list_id IN %s
+                  SELECT mailing_list_id, count(*)
```

Comment:
repassing here, I think we can use the SQL object :) (at some point, we will be forced to use it to make queries)

/odoo/odoo/tools/sql.py